src/yoroi-wallets/cardano/transactionManager.test.ts

```diff
@@ -0,0 +1,86 @@
+import DeviceInfo from 'react-native-device-info'
+
+import {storage as rootStorage, YoroiStorage} from '../storage'
+import {Transaction} from '../types'
+import {mockApi, mockedAddressesByChunks, mockedBackendConfig, mockTx} from './mocks'
+import {makeMemosManager, makeTransactionManager} from './transactionManager'
+
+jest.mock('./api', () => mockApi)
+
+describe('transaction manager', () => {
+  DeviceInfo.getVersion = () => '9.9.9'
+
+  afterEach(rootStorage.clear)
```

Comment:
beforeEach(() => AsyncStorage.clear())

Reply:
The storage needs to be cleared after the last test because it's going to be used in the memo manager tests.

Reply:
Or clear the storage before starting the memos manager tests

Reply:
Using ’afterAll’ should be fine too

Reply:
clear the storage before each of the memo tests too

Reply:
I prefer beforeEach because it's defensive and protects the tests in case someone else forgot to clear